mulighetsrommet-arena-adapter/src/main/kotlin/no/nav/mulighetsrommet/arena/adapter/plugins/Authentication.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.mulighetsrommet.arena.adapter.plugins
+
+import com.auth0.jwk.JwkProviderBuilder
+import io.ktor.server.application.*
+import io.ktor.server.auth.*
+import io.ktor.server.auth.jwt.*
+import no.nav.mulighetsrommet.arena.adapter.AuthConfig
+import java.net.URI
+import java.util.concurrent.TimeUnit
+
+fun Application.configureAuthentication(
+    auth: AuthConfig
+) {
+    val (azure) = auth
+
+    val jwkProvider = JwkProviderBuilder(URI(azure.jwksUri).toURL())
+        .cached(5, 12, TimeUnit.HOURS)
```

Comment:
Av ren nysgjerrighet, hva betyr 5 og 12 her?

Reply:
5 er cache size og 12 timer er expires in ⌛